front/app/utils/participationMethodUtils.tsx

```diff
@@ -38,6 +67,16 @@ const ideationConfig: ParticipationMethodConfig = {
 
 const nativeSurveyConfig: ParticipationMethodConfig = {
   formEditor: 'surveyEditor',
+  onFormSubmission: (project: IProjectData, _ideaId, _idea, _phaseId) => {
+    const urlParameters = `?show_modal=true`;
```

Comment:
We could use this directly without specifying a const since it isn't reused anywhere